dom/observable/tentative/observable-inspect.any.js

```diff
@@ -381,32 +382,39 @@ test(() => {
   });
 
   const results = [];
+  const error = new Error("error from inspect() subscribe handler");
 
   const result = source.inspect({
     abort: () => {
-      results.push('abort() handler run');
-      throw new Error("error from inspect() subscribe handler");
+      results.push("abort() handler run");
+      throw error;
     },
   });
 
   const controller = new AbortController();
 
-  self.when('error').take(1).subscribe(e =>
-      results.push(e.message + ', from report exception'));
+  self.when("error").take(1).subscribe((e) => {
+    results.push("from report exception");
+    results.push(e.error);
```

Comment:
`e` is an [`ErrorEvent`](https://html.spec.whatwg.org/multipage/webappapis.html#the-errorevent-interface), but we want to assert against the `error` constant, so we push `e.error`, which _should_/_does_ point to the `error` constant.